bgpd/bgp_network.c

```diff
@@ -484,10 +484,25 @@ static void bgp_accept(struct event *thread)
 			/* Dynamic neighbor has been created, let it proceed */
 			connection1->fd = bgp_sock;
 
+			if (bgp_set_socket_ttl(connection1) < 0) {
+				peer1->last_reset = PEER_DOWN_SOCKET_ERROR;
+				if (bgp_debug_neighbor_events(peer1))
```

Comment:
debug guard is not needed in such a case when zlog_err()...

Reply:
fixed